pkg/configuration/configuration.go

```diff
@@ -66,6 +66,10 @@ const (
 	// DefaultTestingMode specifies whether the services should run in testing mode.
 	DefaultTestingMode = false
 
+	varE2ETestingMode = "e2etestingmode"
```

Comment:
all const string following diff format .. for ex, log.json, graceful_timeout, http.read_timeout
we should have same format for all of including this `e2etestingmode` .. this is also diff format where words are not separated with any delimiter.

Reply:
@nurali-techie i didn't set this convention - just followed what was already in the file... (specifically was following "testingmode" in any case this will be removed shortly...

Reply:
fixed: e7b06a11435cad5ff5465e0a8b5c62f842d66d88